.github/workflows/ci-it.yaml

```diff
@@ -32,9 +32,15 @@ jobs:
     strategy:
       fail-fast: true
     steps:
-      - uses: actions/checkout@v1
-        with:
-          submodules: true
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          # If your submodules are configured to use SSH instead of HTTPS please uncomment the following line
+          # git config --global url."https://github.com/".insteadOf "git@github.com:"
```

Comment:
What these line for? It is confusing me, as we are using https://github.com

Reply:
I think we only need `git submodule sync --recursive`

Reply:
> `v2` is not compatible with `v1`, at least the parameters are not compatible, and I suggest you fix this in the pull request where you encounter the issue, which can verify the fix works

Sorry for not being clearly marked. It is true that the current V2 does not support the submodule function. Later, try to complete the verification in the local environment before submitting the PR.

Reply:
> What these line for? It is confusing me, as we are using https://github.com

IMO, current is use https, use ssh mean agent by ssh command, such as ssh-agent.
Maybe my understanding is wrong, I did n’t study this deeply.

Reply:
> I think we only need `git submodule sync --recursive`

`git submodule update` updates the contents of the submodules, `git submodule sync` only update metainfo.  FYI  [https://stackoverflow.com/questions/45678862/git-submodule-update-vs-git-submodule-sync/45679230](url)

Reply:
You misunderstand my point. These are comments, what are you trying to say? Is this IF making any sense to anyone? Why We keep this?

Reply:
get it, There is indeed a problem with the comment